.travis.yml

```diff
@@ -1,6 +1,8 @@
+sudo: false
```

Comment:
http://docs.travis-ci.com/user/workers/container-based-infrastructure/#Routing-your-build-to-container-based-infrastructure